app/keepers/keepers.go

```diff
@@ -657,6 +685,7 @@ func initParamsKeeper(appCodec codec.BinaryCodec, legacyAmino *codec.LegacyAmino
 	// whole usage of params module
 	paramsKeeper.Subspace(ibcexported.ModuleName)
 	paramsKeeper.Subspace(ibctransfertypes.ModuleName)
+	paramsKeeper.Subspace(tokenfactorytypes.ModuleName)
```

Comment:
do you think this line is needed ? It seems the current version of token factory stores the params inside the module itself - https://github.com/babylonlabs-io/tokenfactory/blob/main/x/tokenfactory/keeper/params.go ?

Reply:
ahh weird 🤔 then lets leave it 👍